locale/en_US/translations.ts

```diff
@@ -942,6 +946,26 @@
             <source>Resumable</source>
             <translation>Resumable</translation>
         </message>
+        <message>
+            <source>Guide</source>
+            <translation>Guide</translation>
+            <extracomment>Trickplaybar button, Guide</extracomment>
```

Comment:
```suggestion
            <extracomment>Trickplaybar button text</extracomment>
```

No need to add "guide". It's obvious from context